### PR TITLE
Adds liquidity demo tutorial

### PR DESCRIPTION
**Pull request by toschdev**

### CODE REVIEW

File: liquidity-module/index.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+starport relayer configure
+```
+
+For the local `source` chain, use the default values.
+For the testnet `target` chain, use the following values.
+
+
+Target RPC: [https://rpc.testnet.cosmos.network:443](https://rpc.testnet.cosmos.network/)
+
+Token Faucet: [https://faucet.testnet.cosmos.network:443](https://faucet.testnet.cosmos.network/)
```
Comment (alijnmerchant21) [inline liquidity-module/index.md]: Token Faucet is same as Target Faucet?

File: liquidity-module/index.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+From the terminal output that `starport serve` created for you, use the `username` accounts address and claim token from the faucet
+
+```markdown
+curl -X POST -d '{"address": "cosmosxxxxx"}' https://faucet.testnet.cosmos.network
+```
+
+When you get a success message, you can check your balance.
+
+*See balance* [https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/](https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxxxx)
```
Comment (alijnmerchant21) [inline liquidity-module/index.md]: Can we add a tip or something that cosmosxxx should be replaced by your address? Or is it very obvious?

**Review by barriebyron — APPROVED** [review]

File: .vuepress/config.js [hunk 3]
```diff
@@ -84,6 +84,11 @@ module.exports = {
               path: "/connecting-to-testnet/",
               directory: true,
             },
+            {
+              title: "Understanding the Liquidity Module",
```
Comment (barriebyron) [inline .vuepress/config.js:88]:
```suggestion
              title: "About the Liquidity Module",
```

File: liquidity-module/index.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+- Connect your blockchain to the testnet
+- Send your own token with IBC to the testnet
+- Create a pool with your token
+- Use the pool with your token
+
+## Prerequisites
+
+This tutorial uses [Starport](https://github.com/tendermint/starport) v0.16.0. The Starport tool is the easiest way to build a blockchain. 
+
+- [Install Starport](../starport/index.md) v0.16.0 <!-- link to the new tutorial file for this prereq to install Starport https://github.com/cosmos/sdk-tutorials/pull/694/commits/9a988d64408df16dad61412b7c542f6dd1fa4bee  -->
```
[the remark this answers is not in the document]
Comment (toschdev) [reply liquidity-module/index.md]: it is merged

File: liquidity-module/index.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+cd myblockchain
+```
+
+## Add Your Token in the Configuration
+
+*Note: The minimum reserve coin amount for a pool in the liquidity module is 1,000,000. Make sure you create enough tokens for your liquidity pools.*
+
+Open the `config.yml` file in the top-level folder of your app directory. 
+
+For the `accounts` parameter, add your username and your new token:
```
[the remark this answers is not in the document]
Comment (toschdev) [reply liquidity-module/index.md:65]: When it is one token (even with a lot of "coins") I would say singular, when we add more than 1 token, then we would use plural.

File: liquidity-module/index.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+
+```yml
+accounts:
+	- name: username
```
[the remark this answers is not in the document]
Comment (toschdev) [reply liquidity-module/index.md]: It should be 2 spaces for `-name: username` and 4 spaces for `coins: ...`

File: liquidity-module/index.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+Scaffold a new blockchain called `myblockchain`:
+
+```bash
+starport app github.com/username/myblockchain
+cd myblockchain
+```
+
+## Add Your Token in the Configuration
+
+**Note** The minimum reserve coin amount for a pool in the liquidity module is 1,000,000. Make sure you create enough tokens for your liquidity pools.
```
[the remark this answers is not in the document]
Comment (toschdev) [reply liquidity-module/index.md:61]: yes, `minimum: 1000000`

File: .vuepress/config.js (hunk 3 above)
Comment (toschdev) [reply .vuepress/config.js:88]: I do like `Understanding`

File: liquidity-module/index.md [hunk 8]
```diff
@@ -0,0 +1,291 @@
...
+    gaiad version
+    ```
+
+    The output of `gaiad version` should print something like
+    
+    ```bash
+    gravity-dex-fa647b0fefe5508e9c975b3d4f095db2d3d20a13
+    ```
+
+## Create the Blockchain
```
Comment (alijnmerchant21) [inline liquidity-module/index.md:50]: Have we removed the steps for deleting previous blockchain scaffolding ?

Can't seem to find it.

**Pull request merged**